### PR TITLE
♻️ Switch to App Router 

### PR DESCRIPTION
**Pull request by JackDevAU**
## This commit moves away from the Pages Router to the App Router

Changes

- Removes the `pages` folder
- Creates the `app` folder
- This example uses `generateStaticParams` to make the dynamic pages static by default. 

>[!NOTE]  
>To change this to server rendering, make the function async and get the content via the `client.queries.xx`

### CODE REVIEW

File: content/posts/june/June-Blog-Post.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (ncn-ssw) [inline content/posts/june/June-Blog-Post.mdx:1]: **question**: Was this file intended to be included, given that there was no previous equivalent?

Comment (JackDevAU) [reply content/posts/june/June-Blog-Post.mdx:1]: Yes! This is just a demo of the nested pages. I guess we could add better content for it but I think it at the very least demos it working

**Review by ncn-ssw — APPROVED** [review]

File: components/icon.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import * as React from "react";
-import { ColorPickerInput } from "../../tina/fields/color";
-import { IconPickerInput } from "../../tina/fields/icon";
-import { useTheme } from "../layout";
+"use client";
 import * as BoxIcons from "react-icons/bi";
+// import { useLayout } from "./layout/layout-context";
```
Comment (ncn-ssw) [inline components/icon.tsx:3]: **nitpick**: Comment should be removed.

Comment (JackDevAU) [reply components/icon.tsx]: Done!
